Mathlib/CategoryTheory/Monoidal/ExternalProduct/KanExtension.lean

```diff
@@ -0,0 +1,144 @@
+/-
+Copyright (c) 2025 Robin Carlier. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Robin Carlier
+-/
+import Mathlib.CategoryTheory.Monoidal.ExternalProduct.Basic
+import Mathlib.CategoryTheory.Functor.KanExtension.Pointwise
+import Mathlib.CategoryTheory.Limits.Final
+
+/-!
+# Preservation of pointwise left Kan extensions by external products
+
+We prove that if a functor `H' : D' ⥤ V` is pointwise left Kan extended from
```

Comment:
```suggestion
We prove that if a functor `H' : D' ⥤ V` is a pointwise left Kan extended of
```

Reply:
Right, it’s true that this terminology seems to be only for extensions along full subcategories ([this is where I’m taking it from], with an extra "pointwise" since in Kerodon non-pointwise extensions are not a thing)(https://kerodon.net/tag/02Y2)